core/trino-main/src/main/java/io/trino/operator/scalar/DateTimeFunctions.java

```diff
@@ -370,6 +375,16 @@ public static Slice dateFormat(ISOChronology chronology, Locale locale, long tim
     @SqlType("timestamp(3)") // TODO: increase precision?
     public static long dateParse(ConnectorSession session, @SqlType("varchar(x)") Slice dateTime, @SqlType("varchar(y)") Slice formatString)
```

Comment:
Could you add a jmh benchmark for this similar to other benchmarks here e.g. `io.trino.operator.scalar.BenchmarkArrayEqualOperator` and post results before/after in the commit message and in the pr description?

Reply:
done